# request
string model_name
float32 r
float32 p
float32 y
---
# response
bool success
